crash/subsystem/filesystem/mount.py

```diff
@@ -126,6 +126,15 @@ def mount_super(mnt):
         return sb
 
     @export
+    @staticmethod
```

Comment:
This should be handled as a callback where we have two versions, one that returns mnt['mnt_root'] and the other that returns mnt['mnt']['mnt_root'].  See crash.types.task.pick_get_rss for an example.

Reply:
Hmm, ok that does look a bit better.  I'll send you another pull request later, thanks